manual/src/threelang/tfchain/balance.md

```diff
@@ -0,0 +1,40 @@
+# Balance
+
+## Transfer 
+Transfer balance between tfchain twins
+- action name: !!tfchain.balance.transfer
```

Comment:
For balance KDS had some other examples. Please check the examples from https://github.com/threefoldtech/home/issues/1434. There you'll see:
```
!!web3gw.money.send 
    channel:btclight
    from:mywallet
    to:'dfdfdfgdfdfdfdf'
    currency:btc
    amount:0.01
```

So here we should have something like:
```
!!web3gw.money.send 
    channel:tfchain
    from:mytwinid
    to:othertwinid
    currency:tft
    amount:0.01
```